src/app.js

```diff
@@ -22,17 +26,31 @@ export function app(app) {
     })
   }
 
-  emit("ready", render(state, view))
+  render()
+
+  emit("init")
 
   return emit
 
-  function render(state, view) {
-    element = patch(
-      root,
-      element,
-      node,
-      (node = emit("render", view)(state, actions))
-    )
+  function render() {
+    if (!locked) {
+      locked = !locked
+
+      requestAnimationFrame(function() {
+        element = patch(
+          root,
+          element,
+          node,
+          (node = emit("render", view)(state, actions))
+        )
+
+        locked = !locked
```

Comment:
@SkaterDad @ngryman Thank you both! This discussion made looked at this again, but I guess we all now can see that unlocking after emitting `loaded` would prevent actions called inside `loaded` to be schedule a render.

As a rule of thumb, I would use `init` to call actions and not `loaded`, since at that point the DOM has already been patched once, but it should be possible.